### ISSUE TITLE
[Quest][WoTLK][Zul'Drak] Troll Patrol (Daily)

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4648]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Once you do the main quest it should be awarded as a daily. The dailiy is offered 3 times.
**Expected behaviour**: Tell us what should happen instead
Do main quest get quest for daily
**Steps to reproduce the problem**:

1. Do main quest
2. Turn In
3. Get 3 dailies.

**Include proofs for this behaviour**
![wowscrnshot_032917_165524](https://cloud.githubusercontent.com/assets/12252441/24476223/87075664-14a0-11e7-997c-47238865ccda.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://www.wowhead.com/quest=12587/troll-patrol
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TheRealestMcCoy** [comment]
All 3 can be picked up.
![wowscrnshot_032917_165641](https://cloud.githubusercontent.com/assets/12252441/24476282/b3e94b74-14a0-11e7-947c-2eb2698d22fa.jpg)

**Comment by TheRealestMcCoy** [comment]
Seems all 3 can be completed as well by doing each 1 time
![wowscrnshot_032917_170622](https://cloud.githubusercontent.com/assets/12252441/24476642/1fdb759a-14a2-11e7-9474-d65be9ac6152.jpg)

**Comment by Rushor** [comment]
DB/Quest: Troll Patrol (Daily) -- Add Pooling for Daily quests

* also add Banner Visual for Quest: Troll Patrol: High Standards
* also add missing gameobjects for Quest: Troll Patrol: Can You Dig it?
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4648
* commit: ba2436def407130bd8d0025394610fff1b81c241

![GitHub Logo](http://i.imgur.com/H0cWacw.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Feedback for this report from PTR:

>  - Add Pooling for Daily quests

Works as intended.

>  - also add Banner Visual for Quest: Troll Patrol: High Standards

Works as intended.

 > - also add missing gameobjects for Quest: Troll Patrol: Can You Dig it?

Works as intended, however the creatures that spawn https://wowgaming.altervista.org/aowow/?npc=28345 does not attack the player that summoned him and the GO visually despawns when the player moves.

![wowscrnshot_102417_110701](https://user-images.githubusercontent.com/6797611/31931621-72f94dc0-b8ac-11e7-8ae9-6f04735dabb2.jpg)
